lib/etl/transform/transform_json.rb

```diff
@@ -0,0 +1,38 @@
+require 'json'
+require 'json-schema'
+
+class TransformJson
+  def process(input_json)
+    @input_json = input_json
+    pl = parsed_list
+    JSON::Validator.validate(schema_fname, pl)
```

Comment:
Rather than assign the variable `pl`, you can actually just use `parsed_list` here and in the line below.